src/main/java/com/grafana/opentelemetry/GrafanaProperties.java

```diff
@@ -148,12 +146,6 @@ public static class OnPremProperties {
          */
```

Comment:
Line 142 still refers to gRPC

> You do not need to set an <code>endpoint</code> value if your Grafana Agent is running locally
> with the default gRPC endpoint (localhost:4317).

I image this is still true provided it's the http endpoint (`localhost:4318`)